packages/in_app_purchase/in_app_purchase_storekit/pubspec.yaml

```diff
@@ -31,6 +31,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   json_serializable: ^6.0.0
+  pigeon: ^11.0.1
```

Comment:
`pigeon` is currently at version `16`. Could you regenerate at the latest version? Version 13 adds good quality of like changes for Objective-C: https://pub.dev/packages/pigeon/changelog#1300, so it will probably prevent your code from compiling without changes.

Reply:
Nice catch @bparrishMines, this seems like the sort of thing that we should be checking in CI.